appengine-push/README.md

```diff
@@ -44,7 +44,7 @@ TODO(tmatsuo): Better implementation for devserver.
     - Replace 'your-application-id' with your real application id.
 
 - Edit constants.py
-    - Replace '{AN_UNIQUE_TOKEN}' with your random unique token.
+    - Replace '{AN_UNIQUE_TOKEN}' with an arbitrary string of your choosing.
```

Comment:
"an arbitrary secret string of your choice to protect the endpoint from abuse"
?